kernelcare/assets/configuration/spec.yaml

```
@@ -0,0 +1,28 @@
+name: kernelcare
+files:
+- name: kernelcare.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: key
+      required: false
+      description: |
+        Kernelcare license key
+      value:
+        type: string
+    - name: login
+      required: false
+      description: |
+        Kernelcare login
+      value:
+        type: string
+    - name: api_token
+      required: false
+      description: |
+        Kernelcare API security token
+      value:
+        type: string
+    - template: instances/default
```

```suggestion
    - template: instances/default
    - template: instances/http
```
can consider including the configuration options that the http wrapper supports

---

added